LICENSE

```diff
@@ -1,4 +1,4 @@
-Built by the WHO COVID App Collective (see content/credits.yaml).
+Built by the WHO COVID-19 App Collective (see content/credits.yaml).
```

Comment:
does the switch to COVID-19 apply here too?

Reply:
our credits screen says COVID-19 - should use one name